src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/BLEU/BLEUAlgorithm.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI.Evaluation.NLP.BLEU;
+
+internal static class BLEUAlgorithm
+{
+    internal static IEnumerable<NGram<T>> NGrams<T>(IEnumerable<T> input, int n)
```

Comment:
This seems identical to the `NGram.Create` function in NGram.cs which was subsequently converted into a `CreateNGrams` extension method. Can this be deleted in favor of that extension method?